src/services/offlineService.test.ts

```diff
@@ -36,6 +37,7 @@ describe("Offline Service", () => {
     const writeFileSpy = jest.spyOn(fs, "writeFileSync");
     await service.build();
     const calls = writeFileSpy.mock.calls;
+    writeFileSpy.mockRestore();
```

Comment:
why do we need to restore here and at the end of the test?

Reply:
We saw this a bunch in VoTT as well. For some reason, if you don't restore some spies individually, the tests conflict with each other. This makes it more explicit and guarantees that they will be restored.